pipeline-samples/README.md

```diff
@@ -0,0 +1,9 @@
+Contains unbuilt pipeline samples. To build the samples, install `kfp` from
+pypi, then run this command:
+
+    find pipeline-samples/ -name "*.py" -exec bash -c 'dsl-compile --py {} --output charms/pipelines-api/files/$(basename {} .py).yaml' \;
```

Comment:
The rest of the readme is in italics after this `*.py`.  Is that a github render issue, or do you need to escape something here?

Reply:
I'm going to call this a bug in Github's source diff mode, since it looks fine after being rendered as Markdown. It also renders fine on other markdown tools.